Formula/vitess.rb

```diff
@@ -0,0 +1,41 @@
+class Vitess < Formula
+  desc "Database clustering system for horizontal scaling of MySQL"
+  homepage "https://vitess.io"
+  url "https://github.com/vitessio/vitess/archive/v8.0.0.tar.gz"
+  sha256 "c47320b9bcb874b1a6dfca78ec677be7c4bb4c7b2a6470df80bd1bc0ad125e92"
+  license "Apache-2.0"
+
+  depends_on "automake" => :build
+  depends_on "go" => :build
```

Comment:
If go is used, how are the dependencies/modules handled? Where are their versions specified, are they pinned, etc?

Reply:
I'm not super familiar with Go packaging, but looks like `go.sum`: https://github.com/vitessio/vitess/blob/master/go.sum

Reply:
vitess uses go modules, the dependencies are in [go.mod](https://github.com/vitessio/vitess/blob/master/go.mod)